### PR TITLE
PLT-206 Fix first channel load scrolling issue on Windows and iOS and add emoji to RHS.

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

**Comment by hmhealey** [comment]
Looks good to me

File: web/react/components/post_list.jsx [hunk 1]
```diff
@@ -83,6 +83,7 @@ export default class PostList extends React.Component {
         };
     }
     componentDidMount() {
+        window.onload = () => this.scrollToBottom();
```
Comment (crspeller) [inline web/react/components/post_list.jsx]: Why is this necessary? It seems strange.

Comment (jwilander) [inline web/react/components/post_list.jsx]: It is strange, I spent 3 hours trying to do it another way. iOS does this weird thing with React where it waits for everything to render before loading the DOM so this was the only way to scroll at the correct time on a new page load.

I can add a condition to only do it when it's iOS but then if it occurs on other devices we'd have to maintain the condition. Leaving it in for all devices didn't cause any issues I could find

Comment (crspeller) [inline web/react/components/post_list.jsx]: The only issue I can see is strangeness if we where to implement a scroll position remembering feature. But if you have tested it, it's all good.
